### ISSUE TITLE
Questions about running on Newer College dataset

### ISSUE THREAD

**Issue by marrblue** [VIS4ROB-lab/voxfield#11]
Hi @YuePanEdward !

I am currently going to run voxfield on Newer College dataset for comparison with other methods.
The ncd examble data is downloaded from shine-mapping repo.
Next I use [kitti2bag](git@github.com:tomas789/kitti2bag.git) transform the pointcloud data and poses into a rosbag.
![image](https://github.com/VIS4ROB-lab/voxfield/assets/31070287/10fdca1b-6194-42cb-a32e-3bfe1bba0859)
but it seems that the transformation process is not working.
The output of voxfield is as follows:
![image](https://github.com/VIS4ROB-lab/voxfield/assets/31070287/db33cbb1-668b-4ddd-b6f3-508c63d20142)

Basically I'm using the same parameters and configuration as for the maicity dataset. 
Could you please provide some hints or share a compatible rosbag file for the ncd example data? Thank you in advance!

**Comment by marrblue** [comment]
I solve this problem by set right frame_id for tf messsage
````
tf_stamped.header.frame_id = 'world'
tf_stamped.child_frame_id = 'velodyne'
````
